--- upstream/contents/config/config.qml	2026-06-02 02:55:55.960627714 +0800
+++ current/contents/config/config.qml	2026-06-02 02:55:55.964627714 +0800
@@ -32,6 +32,11 @@
         source: "configTimeZones.qml"
         includeMargins: false
     }
+    ConfigCategory {
+        name: i18n("Translation")
+        icon: "preferences-desktop-locale"
+        source: "configTranslation.qml"
+    }
 
     readonly property PlasmaCalendar.EventPluginsManager eventPluginsManager: PlasmaCalendar.EventPluginsManager {
         Component.onCompleted: {
