--- upstream/contents/ui/configTimeZones.qml
+++ current/contents/ui/configTimeZones.qml
@@ -19,6 +19,29 @@
 Kirigami.PageRow {
     id: timeZonesRow
 
+    readonly property string upstreamTranslationDomain: "plasma_applet_org.kde.plasma.digitalclock"
+    function i18n(message, arg1, arg2, arg3) {
+        if (arguments.length === 4) {
+            return i18nd(upstreamTranslationDomain, message, arg1, arg2, arg3);
+        }
+        if (arguments.length === 3) {
+            return i18nd(upstreamTranslationDomain, message, arg1, arg2);
+        }
+        if (arguments.length === 2) {
+            return i18nd(upstreamTranslationDomain, message, arg1);
+        }
+        return i18nd(upstreamTranslationDomain, message);
+    }
+    function i18nc(context, message, arg1, arg2) {
+        if (arguments.length === 4) {
+            return i18ndc(upstreamTranslationDomain, context, message, arg1, arg2);
+        }
+        if (arguments.length === 3) {
+            return i18ndc(upstreamTranslationDomain, context, message, arg1);
+        }
+        return i18ndc(upstreamTranslationDomain, context, message);
+    }
+
     property string title
     property string cfg_lastSelectedTimezone
     property alias cfg_selectedTimeZones: timeZones.selectedTimeZones
