--- upstream/contents/ui/configCalendar.qml
+++ current/contents/ui/configCalendar.qml
@@ -19,6 +19,26 @@
 KCMUtils.SimpleKCM {
     id: calendarPage
 
+    readonly property string upstreamTranslationDomain: "plasma_applet_org.kde.plasma.digitalclock"
+    function i18n(message, arg1, arg2) {
+        if (arguments.length === 3) {
+            return i18nd(upstreamTranslationDomain, message, arg1, arg2);
+        }
+        if (arguments.length === 2) {
+            return i18nd(upstreamTranslationDomain, message, arg1);
+        }
+        return i18nd(upstreamTranslationDomain, message);
+    }
+    function i18nc(context, message, arg1, arg2) {
+        if (arguments.length === 4) {
+            return i18ndc(upstreamTranslationDomain, context, message, arg1, arg2);
+        }
+        if (arguments.length === 3) {
+            return i18ndc(upstreamTranslationDomain, context, message, arg1);
+        }
+        return i18ndc(upstreamTranslationDomain, context, message);
+    }
+
     signal configurationChanged()
 
     property alias cfg_showWeekNumbers: showWeekNumbers.checked
